covalent/_workflow/lattice.py

```diff
@@ -321,6 +323,7 @@ def lattice(
     deps_pip: Union[DepsPip, list] = None,
     call_before: Union[List[DepsCall], DepsCall] = [],
     call_after: Union[List[DepsCall], DepsCall] = [],
+    triggers: Union["BaseTrigger", List["BaseTrigger"]] = None,
```

Comment:
Can't you use the BaseTigger object you imported in the TYPE_CHECKING import section instead of the string?

Reply:
That is how type hinting works when we are conditionally importing things. So for example, in case `TYPE_CHECKING` is False, this line (326) is still run and will throw an error. Which is why `TYPE_CHECKING` is used along with the strings of the objects imported inside of it to have type checking without any errors.